### PR TITLE
Add a management command to raise a test exception.

### LINKED ISSUE #1503: [Suggestion] Create a management command to raise a simple exception for testing error reporters
I'd love to add a simple management command that raises an exception like:

```
raise Exception("This is a test exception by django-extensions.")
```

It would make testing my error reporting software a lot easier. Would a PR for this be welcomed?

Thanks as always for maintaining this library!!!

### PR DESCRIPTION
**Pull request by YPCrumble**
Fixes #1503.

### CODE REVIEW

**Comment by YPCrumble** [comment]
@trbs did CI stop working recently? It looks like the recent PR's don't have TravisCI checks.

File: tests/management/commands/test_raise_test_exception.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from django.core.management import call_command
+
+
+def test_raise_test_exception():
+    exception_raised = False
+    try:
+        call_command('raise_test_exception')
+    except Exception:
```
Comment (yeojin-dev) [inline tests/management/commands/test_raise_test_exception.py:9]: Please consider using `with self.assertRaises()` code in your work. :)

Comment (YPCrumble) [reply tests/management/commands/test_raise_test_exception.py:9]: @yeojin-dev notice there isn’t a `self` argument here - I didn’t use it to follow the structure of other tests in the repo. If it makes sense to use django’s `TestCase` here I’d be happy to do it, I just wasn’t sure why it wasn’t used elsewhere.

**Comment by trbs** [comment]
> @trbs did CI stop working recently? It looks like the recent PR's don't have TravisCI checks.

Something seems to have broken with Travis CI or GitHub :(
So far have not found what is causing this... maybe these types of reports are related https://twitter.com/a_grebenyuk/status/1254823757778087936 but also do not have an answer...

**Comment by trbs** [comment]
https://travis-ci.community/t/github-status-not-posted-on-commits-on-repositories-using-legacy-service-integration/7798

File: tests/management/commands/test_raise_test_exception.py (hunk 1 above)
Comment (trbs) [reply tests/management/commands/test_raise_test_exception.py:9]: Or maybe:

```python
with pytest.raises(SomeException):
    call_command('raise_test_exception')
```

Comment (YPCrumble) [reply tests/management/commands/test_raise_test_exception.py]: @trbs great suggestion - I just updated the code and created a `DjangoExtensionsTestException` Exception subclass.

**Pull request merged**